commit 33ef4797685679211ecc9438bfa357b0fcf48ccf
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Wed Oct 31 14:56:59 2018 +0000

    media: tc358743: Increase FIFO level to 374.
    
    The existing fixed value of 16 worked for UYVY 720P60 over
    2 lanes at 594MHz, or UYVY 1080P60 over 4 lanes. (RGB888
    1080P60 needs 6 lanes at 594MHz).
    It doesn't allow for lower resolutions to work as the FIFO
    underflows.
    
    374 is required for 1080P24-30 UYVY over 2 lanes @ 972Mbit/s, but
    >374 means that the FIFO underflows on 1080P50 UYVY over 2 lanes
    @ 972Mbit/s.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/media/i2c/tc358743.c b/drivers/media/i2c/tc358743.c
index 26070fb6ce4e..3113e56c884b 100644
--- a/drivers/media/i2c/tc358743.c
+++ b/drivers/media/i2c/tc358743.c
@@ -1946,7 +1946,7 @@ static int tc358743_probe_of(struct tc358743_state *state)
 	state->pdata.ddc5v_delay = DDC5V_DELAY_100_MS;
 	state->pdata.enable_hdcp = false;
 	/* A FIFO level of 16 should be enough for 2-lane 720p60 at 594 MHz. */
-	state->pdata.fifo_level = 16;
+	state->pdata.fifo_level = 374;
 	/*
 	 * The PLL input clock is obtained by dividing refclk by pll_prd.
 	 * It must be between 6 MHz and 40 MHz, lower frequency is better.
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1339_33ef4797685679211ecc9438bfa357b0fcf48ccf.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1339_33ef4797685679211ecc9438bfa357b0fcf48ccf.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
